toolsets/cmake/ham-cmake

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
```

Comment:
How do you build with ham-cmake?

Is it possible to tell it to output .lib / public headers, etc... in a specific folder?

Could it keep the build stuff in ~/_ham/something and copy the .libs, etc... in the standard niLang/libs/ folder ?